var/spack/repos/builtin/packages/msmpi/package.py

```diff
@@ -41,29 +41,37 @@ def determine_version(cls, exe):
         ver_str = re.search(r"Microsoft MPI Startup Program \[Version ([0-9.]+)\]", output)
         return Version(ver_str.group(1)) if ver_str else None
 
+    def setup_dependent_package(self, module, dependent_spec):
+        spec = self.spec
+
+        spec.mpicc = spack_cc
```

Comment:
Normally these would be assigned to whatever compiler wrappers come with the MPI installation. For `msmpi`, are there no such wrappers?

As it is, this is any dependent package is being told that the MPI compiler wrappers should just be whatever compiler that dependent package happens to be using. If that's the intent, IMO a comment would be useful here making that explicit (and pointing out that this is not typical).

(all the other changes make sense to me)

Reply:
AFAICT from the docs for building a project with msmpi (and a number of projects I've ported on Windows) msmpi does not have wrappers, you just specify the requisite includes and libs to the compiler/linker manually (so just manual wrappers), which is a lot of what the wrappers do anyway. 

I'll update the comments to reflect this.